app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<header class="auth">
+  <div class="main-header-col">
+    <h1>Ometman Budget App</h1>
+    <p>Welcome!! Login or Register to use</p>
+  </div>
+</header>
+<main class="auth">
+<h2 class="login-heading"> Log in </h2>
+  <div class="login-container">
+    <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: 'login-container-form' }) do |f| %>
+      <div class="field login-field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'login-input' %>
+      </div>
+
+      <div class="field login-field">
+        <%= f.label :password %><br />
+        <%= f.password_field :password, autocomplete: "current-password", class: 'login-input' %>
+      </div>
+
+      <% if devise_mapping.rememberable? %>
+        <div class="field login-field">
+          <%= f.check_box :remember_me %>
+          <%= f.label :remember_me %>
+        </div>
+      <% end %>
+
+      <div class="actions login-actions">
+        <%= f.submit "Log in", class: 'login-button' %>
+      </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] It would be nice if you could work on the styles of the Login form to match the original design as it makes the UI look good and improves the user experience. Kindly align the labels to the left instead of center alignment. 👍🏽

**Your Design**

<img width="443" alt="Screenshot 2024-02-10 at 6 20 55 PM" src="https://github.com/ometman/budget-app/assets/50941074/d83774c6-16ef-4662-b911-3fb62345ca45">

**The Original Design**

<img width="341" alt="Screenshot 2024-02-10 at 6 38 42 PM" src="https://github.com/ometman/budget-app/assets/50941074/83c64ea3-09a9-4f58-8596-4897624820f0">